### PR TITLE
ROC-6050: Update logic for calculating pilot court

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[ROC-6050](https://tools.hmcts.net/jira/browse/ROC-6050)

### Change description ###
This PR updates the logic for pilot courts

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

### CODE REVIEW

File: domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/PilotCourt.java [hunk 1]
```diff
@@ -18,7 +19,10 @@ public String getName() {
     }
 
     public static boolean isPilotCourt(String courtName) {
+        if (courtName == null) {
+            return false;
+        }
         return Arrays.stream(PilotCourt.values())
-            .anyMatch(val -> val.name().equalsIgnoreCase(courtName));
+            .anyMatch(pilotCourt -> courtName.toLowerCase().contains(pilotCourt.name));
```
Comment (markry-kainos) [inline domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/PilotCourt.java]: is contains ok? should it be equals? what if we have something like north birmingham.

Comment (sabahirfan) [reply domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/PilotCourt.java]: That's exactly what the new logic is. 
```
It is accepted as pilot court if any of the below text is there in the suggested court name

- Manchestor
- Edmonton
- Birmingham
- Clerkenwell 
- Shoreditch

```

Comment (markry-kainos) [reply domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/PilotCourt.java]: cool 👍 😄

Comment (simad) [inline domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/PilotCourt.java]: _eyetwitch_

**Pull request merged**